webapp/src/components/channel_header_dropdown_button/index.ts

```diff
@@ -26,7 +26,7 @@ const mapStateToProps = (state: GlobalState) => {
         hasCall: currentChannelHasCall(state),
         isAdmin: isCurrentUserSystemAdmin(state),
         isCloudStarter: isCloudStarter(state),
-        isCloudPaid: isCloudProfessionalOrEnterpriseOrTrial(state),
+        isCloudPaid: isCloudProfessionalOrEnterpriseorEnterpriseAdvanceOrTrial(state),
```

Comment:
Wondering if these checks are still needed for cloud? Could be out of scope but wondering if we can simplify these and just check for trials @cpoile @streamer45

Reply:
I'd leave these be for now because they would need changes on Mobile as well I think. We'll review them eventually.